src/lib/util/aviio.cpp

```diff
@@ -3575,71 +3557,70 @@ avi_file::error avi_file_impl::soundbuf_flush(bool only_flush_full)
  * @param   indent              The indent.
  */
 
-void avi_file_impl::printf_chunk_recursive(avi_chunk const *container, int indent)
+void avi_file_impl::display_chunk_recursive(avi_chunk const *container, int indent)
```

Comment:
I got rid of the u64toa function as you suggested, but this turned into pulling a thread a bit.
Having `printf` in the function name didn't really make sense, however I can't see where this function is even called.